specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/virtualMachineScaleSet.json

```diff
@@ -3925,6 +3931,18 @@
             "lun"
           ],
           "description": "Specifies the parameters that are used to add data disks to the virtual machines in the scale set. <br><br> For more information about disks, see [About disks and VHDs for Azure virtual machines](https://docs.microsoft.com/azure/virtual-machines/managed-disks-overview)."
+        },
+        "diskControllerType": {
+          "type": "string",
+          "description": "Specifies the DiskController Type",
```

Comment:
Specifies the disk controller type configured for the virtual machines in the scale set. This property is only supported for virtual machines whose operating system disk and and VM size supports hyperVGeneration: V2.
For more information about Disk Controller Types supported please refer to https://aka.ms/azure-diskcontrollertypes.

Reply:
Done